projects/plugins/crm/src/automation/class-automation-engine.php

```diff
@@ -14,23 +14,23 @@
  */
 class Automation_Engine {
 
-	/** @var Automation_Engine Instance singleton */
+	/** @var Automation_Engine Instance singleton. */
```

Comment:
WP Coding standards expect a summary and description in different lines, in the form:
```
/**
 * Summary.
 *
 * @var type $var Description.
 */
```
Source: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#2-1-class-members

Reply:
Fixed!